src/links/v1/providers.json

```diff
@@ -65,9 +65,9 @@
       "id": "mp",
       "attributes": {
         "name": "materialsproject.org",
-        "description": "",
-        "base_url": null,
-        "homepage": null
+        "description":  "The Materials Project: A database of computed materials properties for the discovery and design of novel materials.",
```

Comment:
I would suggest to move the "title" here to `"name"`. (I intend to do the same for Materials Cloud.)